### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by bigmosi**
I used both Grid and flexbox to position content on the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,47 +34,48 @@ <h1>Software Developer</h1>
...
-<div class="sub-title">
-    <p class="selenium">Selenium</p>
-</div>
-<div class="skill">
-    <p class="inner-head">Skills</p>
-    <div class="inner-title">
+  <ul class="sub-title">
+      <li class="paragraph1">Language</li>
+      <li class="paragraph1">Frameworks</li>
+  </ul>
```
Comment (AnselemOdims) [inline index.html:40]: - [ ] Please ensure make sure your design looks very closely to the original design and adheres to its guidelines(fonts, colors, margins, padding etc)
- [ ] As can be seen below, your background color does not match the design background color. Kindly adjust to match the design.
- [ ] The background is also supposed to have images that are not in your design. Please include them to the design
- [ ] The buttons are not designed to match the original design.

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @bigmosi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please make sure you are not deleting an PR before asking for another review
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!. Kindly tag me with @AnselemOdims for a quicker reply or contact me on slack.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @bigmosi  👋,

_This is an additional review as a result of the second opinion request._

You have done a splendid job so far with your portfolio, keep going 👍🏽 

Since you need nothing like clarification about the reviews given, I will say that you fix the button display properties and then submit a CRR whenever you are ready.

> Follow the lead I sent you on slack on how exactly to fix the button properties of pressed, active and disabled. Tag me here if you need more assistance with this implementation.

Keep rocking 🚀!

**Comment by bigmosi** [comment]
All 3 reviews are used up. kindly request for approval

**Review by mke2111 — COMMENTED** [review]
Hello @bigmosi 👋,

_This is an additional review as a result of the second opinion request._

As pointed out by @Davidosky007, the button implementation is well worked on.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

**Comment by bigmosi** [comment]
> Hello @bigmosi wave,
> 
> _This is an additional review as a result of the second opinion request._
> 
> As pointed out by @Davidosky007, the button implementation is well worked on.
> 
> As this was the only reason that your project was not approved it will be approved as soon as possible. Please note that since we will approve it manually so this could take a few hours!
> 
> Keep rocking rocket!

Okay

**Pull request merged**